website/docs/r/directory_service_directory.html.markdown

```diff
@@ -135,13 +145,25 @@ The following arguments are supported:
 * `enable_sso` - (Optional) Whether to enable single-sign on for the directory. Requires `alias`. Defaults to `false`.
 * `type` (Optional) - The directory type (`SimpleAD`, `ADConnector` or `MicrosoftAD` are accepted values). Defaults to `SimpleAD`.
 * `edition` - (Optional) The MicrosoftAD edition (`Standard` or `Enterprise`). Defaults to `Enterprise` (applies to MicrosoftAD type only).
+* `radius_settings` - (Optional) Radius MFA settings (Not available for `SimpleAD`). Fields documented below.
 * `tags` - (Optional) A map of tags to assign to the resource.
 
 **vpc_settings** supports the following:
 
 * `subnet_ids` - (Required) The identifiers of the subnets for the directory servers (2 subnets in 2 different AZs).
 * `vpc_id` - (Required) The identifier of the VPC that the directory is in.
 
+**radius_settings** supports the following:
+
+* `protocol` - (Required) The protocol specified for your RADIUS endpoints.
```

Comment:
lets add the vaild values for this (as this is a closed list) to make it easier on users

Reply:
done